src/transaction/SignatureMap.js

```diff
@@ -37,17 +42,65 @@ export default class SignatureMap extends ObjectMap {
     static _fromTransaction(transaction) {
```

Comment:
If you have time you can add an explanation of what the function does and maybe a little comments what the rowLength and columns represent in terms of signatures, nodes and etc.